test/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests/StackExchangeRedisCallsInstrumentationTests.cs

```diff
@@ -74,71 +75,112 @@ public void SuccessfulCommandTestWithKey(string value)
         Assert.Equal("EVAL", exportedItems[0].DisplayName);
         Assert.Equal("EVAL redis.call('set', ARGV[1], ARGV[2])", exportedItems[0].GetTagValue(SemanticConventions.AttributeDbStatement));
 
-        VerifyActivityData(exportedItems[1], false, connection.GetEndPoints()[0], true);
-        VerifyActivityData(exportedItems[2], true, connection.GetEndPoints()[0], true);
-        VerifyActivityData(exportedItems[3], false, connection.GetEndPoints()[0], true);
+        VerifyOldActivityData(exportedItems[1], false, connection.GetEndPoints()[0], true);
+        VerifyOldActivityData(exportedItems[2], true, connection.GetEndPoints()[0], true);
+        VerifyOldActivityData(exportedItems[3], false, connection.GetEndPoints()[0], true);
         VerifySamplingParameters(sampler.LatestSamplingParameters);
     }
 
     [Trait("CategoryName", "RedisIntegrationTests")]
     [SkipUnlessEnvVarFoundTheory(RedisEndPointEnvVarName)]
-    [InlineData("value1", null)]
-    [InlineData("value1", "serviceKey")]
-    public void SuccessfulCommandTest(string value, string? serviceKey)
+    [InlineData("value1", null, true, false)]
+    [InlineData("value1", null, false, true)]
+    [InlineData("value1", null, true, true)]
+
+    [InlineData("value1", "serviceKey", true, false)]
+    [InlineData("value1", "serviceKey", false, true)]
+    [InlineData("value1", "serviceKey", true, true)]
+
+    public void SuccessfulCommandTest(
+        string value,
+        string? serviceKey,
+        bool emitOldAttributes,
+        bool emitNewAttributes)
     {
+        var stabilityOptIn = (emitOldAttributes, emitNewAttributes) switch
+        {
+            (false, true) => "database",
+            (true, true) => "database/dup",
+            _ => null,
+        };
+        Environment.SetEnvironmentVariable(DatabaseSemanticConventionHelper.SemanticConventionOptInKeyName, stabilityOptIn);
```

Comment:
You could [promote this to a shared test class](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/dc6cf9e2d78d8ab07d405538f62673af4990f003/test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTests.cs#L321-L337) (but needs a tweak to support `database/dup`) and use rather than manually fudge with the env vars.